### PR TITLE
BETA: Register aliwoto.is-a.dev

### PR DESCRIPTION
**Pull request by ALiwoto**
Added `aliwoto.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/aliwoto.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ALiwoto",
+        "email": "aminnimaj@gmail.com"
+    },
+    "record": {
+        "A": ["176.9.175.107"]
```
Comment (Stef-00012) [inline domains/aliwoto.json:7]: I cannot reach your site
"176.9.175.107 refused to connect."

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by ALiwoto** [comment]
@DEV-DIBSTER Can I re-open this pr? I am fixing the issue on my host rn.

**Comment by DEV-DIBSTER** [comment]
@ALiwoto Yes absolutely, I only end up closing these due to almost every single one going inactive in the first 24 hours gets no response.

**Comment by DEV-DIBSTER** [comment]
Just re-opened for ya.

**Comment by ALiwoto** [comment]
Thanks a lot! I got my site up (port 80).

**Review by wdhdev — APPROVED** [review]
Finally someone who responds to requested changes! Thank you for your quick co-operation!

**Pull request merged**